marketplace-ui/src/app/modules/product/product-detail/product-detail.component.ts

```diff
@@ -176,12 +176,21 @@ export class ProductDetailComponent {
     });
   }
 
+  private isPropertyNullOrAllEmpty(property: DisplayValue | null) {
+    if (property === null) {
+      return true;
+    } else {
+      const isValueEmpty = (currentValue: string) => currentValue === '';
+      return Object.values(property).every(isValueEmpty);
+    }
+  }
+
   getContent(value: string): boolean {
     const content = this.productModuleContent();
     const conditions: { [key: string]: boolean } = {
-      description: content.description !== null,
-      demo: content.demo !== null,
-      setup: content.setup !== null ,
+      description: !this.isPropertyNullOrAllEmpty(content.description),
```

Comment:
The issue was not fixed in case of the language of description is empty String
![image](https://github.com/user-attachments/assets/e5f56124-b134-40ab-bd8d-59ee2dc3b351)


![image](https://github.com/user-attachments/assets/d57dc5bc-00fe-422b-b172-67af540e4e3c)

Reply:
Fixed as commented